### PR TITLE
Fix Microsoft Sentinel mirroring BadRequest

### PR DESCRIPTION
**Pull request by mmhw**
## Status
- [ ] In Progress
- [x] Ready
- [ ] In Hold - (Reason for hold)

## Related Issues
fixes: [link to the issue](https://jira-hq.paloaltonetworks.local/browse/XSUP-23398)
fixes: [link to the issue](https://jira-hq.paloaltonetworks.local/browse/XSUP-24565)

## Description
Sent to customer for review

### CODE REVIEW

File: Packs/AzureSentinel/Integrations/AzureSentinel/AzureSentinel.py [hunk 1]
```diff
@@ -509,7 +509,8 @@ def get_modified_remote_data_command(client: AzureSentinelClient, args: Dict[str
         GetModifiedRemoteDataResponse object, which contains a list of the modified incidents IDs.
     """
     remote_args = GetModifiedRemoteDataArgs(args)
-    last_update = remote_args.last_update
+    last_update = dateparser.parse(remote_args.last_update, settings={'TIMEZONE': 'UTC'}).strftime(  # type: ignore
+        '%Y-%m-%dT%H:%M:%S.%fZ')
```
Comment (bziser) [inline Packs/AzureSentinel/Integrations/AzureSentinel/AzureSentinel.py:513]: please make `'%Y-%m-%dT%H:%M:%S.%fZ'` global variable

File: Packs/AzureSentinel/Integrations/AzureSentinel/AzureSentinel_test.py [hunk 2]
```diff
@@ -1505,9 +1505,10 @@ def test_get_modified_remote_data_command(mocker):
     mock_response = {'value': [{'name': 'incident-1'}, {'name': 'incident-2'}]}
     mocker.patch.object(client, 'http_request', return_value=mock_response)
 
-    last_update = '2023-01-06T08:17:09Z'
+    last_update = '2023-01-06T08:17:09.001016488+02:00'
```
Comment (bziser) [inline Packs/AzureSentinel/Integrations/AzureSentinel/AzureSentinel_test.py:1508]: add test for utc also

File: Packs/AzureSentinel/ReleaseNotes/1_5_7.md [hunk 3]
```diff
@@ -0,0 +1,7 @@
+
+#### Integrations
+
+##### Microsoft Sentinel
+
+- Fixed an issue where the ***get-modified-remote-data*** command not working properly with not UTC timezones.
```
Comment (bziser) [inline Packs/AzureSentinel/ReleaseNotes/1_5_7.md:6]:
```suggestion
- Fixed an issue where the ***get-modified-remote-data*** command did not work correctly with non-UTC timezones.
```

**Review by bziser — COMMENTED** [review]
great catch

**Review by thefrieddan1 — APPROVED** [review]

**Pull request merged**